FMSgridtools/shared/gridobj.py

```diff
@@ -86,161 +34,20 @@ def __post_init__(self):
     def from_file(cls, filepath: str) -> "GridObj":
         check_file_is_there(filepath)
         with xr.open_dataset(filepath) as ds:
-            varlist = list(ds.data_vars)
-            if "tile" in varlist:
-                _tile = ds.tile.values.item().decode('ascii')
-            if "x" in varlist:
-                _x = np.ascontiguousarray(ds.x.values)
-            if "y" in varlist:
-                _y = np.ascontiguousarray(ds.y.values)
-            if "dx" in varlist:
-                _dx = np.ascontiguousarray(ds.dx.values)
-            if "dy" in varlist:
-                _dy = np.ascontiguousarray(ds.dy.values)
-            if "area" in varlist:
-                _area = np.ascontiguousarray(ds.area.values)
-            if "angle_dx" in varlist:
-                _angle_dx = np.ascontiguousarray(ds.angle_dx.values)
-            if "angle_dy" in varlist:
-                _angle_dy = np.ascontiguousarray(ds.angle_dy.values)
-            if "arcx" in varlist:
-                _arcx = ds.arcx.values.item().decode('ascii')
             return cls(
```

Comment:
this too, this can be shortened to return cls( dataset=xr.open_dataset(filepath),...)